src/components/cards/PromoCard.tsx

```diff
@@ -49,7 +50,14 @@ export function PromoCard(props: Props) {
           {message.message}
         </EdgeText>
         <TouchableOpacity onPress={handleClose}>
-          <AntDesignIcon name="close" color={theme.iconTappable} size={theme.rem(1)} style={styles.close} />
+          <AntDesignIcon
+            name="close"
+            color={theme.iconTappable}
+            size={theme.rem(1)}
+            style={styles.close}
+            accessibilityHint={lstrings.close_hint}
+            testID={lstrings.close_hint}
```

Comment:
Also can we sort these props?

Reply:
It may be redundant for now but it is a second option that allows for more fine grained element specificty should we need it. Elements with a `testID` attribute are targetable with
```
tapOn:
    id: 'Close'
```
in comparison, when the `accessibilityHint` is set:
```
tapOn: 'Close'
```
or 
```
tapOn:
    text: 'Close'
```
More info re: this behavior is available in the [Selectors](https://maestro.mobile.dev/api-reference/selectors) documentation.